std_msgs/Header header
autoware_perception_msgs/TrafficLightRoi[] rois
